andela/travel_tool_front
# #160815252 Fix notification prop-type warnings

#### What does this PR do?
Fixes notification prop-type warnings

#### Description of Task to be completed?
- add propTypes for `id` in notificationItem component
 - fix some console errors

#### How should this be manually tested?
- Clone the repo
- Checkout to this branch
- Start the server - `make start`
- You should not see prop-type warnings in your console

#### Any background context you want to provide?
None


#### What are the relevant pivotal tracker stories?
[#160815252](https://www.pivotaltracker.com/story/show/160815252)

#### Screenshots (if appropriate)
None

#### Questions:
None
